ui/pages/settings/networks-tab/index.scss

```diff
@@ -226,7 +226,7 @@
   &__footer {
     display: flex;
     flex-flow: row nowrap;
-    margin: 0.75rem 0;
+    padding: 0.75rem 0;
```

Comment:
max-height of this section is set to `465px` I found that in case of Custom RPC there is an overflow. margin does not add to overall height of element but padding does and thus padding-bottom is visible in overflow (some context here https://stackoverflow.com/a/21756251). Thus I replaced margin by padding to fix the issue.

Increasing max-height is also an option to fix it, but that might break the view for other networks. Thus I would avoid it.